src/ui/public/inspector/ui/inspector_panel.js

```diff
@@ -79,14 +79,11 @@ class InspectorPanel extends Component {
   }
 
   render() {
-    const { views, onClose, title } = this.props;
+    const { views, title } = this.props;
     const { selectedView } = this.state;
 
     return (
-      <EuiFlyout
-        onClose={onClose}
-        data-test-subj="inspectorPanel"
-      >
+      <EuiFlyoutBody>
```

Comment:
This should rather be `<React.Fragment>` so we have the header and body on the toplevel of the `EuiFlyout`.